src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -366,6 +370,22 @@ def _service_instance_action(self, cmd):
 
         return HandleCommandResult()
 
+    def _host_ls(self, cmd):
```

Comment:
please copy the code from 

https://github.com/noahdesu/ceph/blob/8e108d198fa8f2fe6b1a152659b00b316df3e31f/src/pybind/mgr/orchestrator_cli/module.py#L163-L179

at the same lines. Let's prevent any non-trivial merge conflicts.

Reply:
done. We can wait @noahdesu to merge and after that rebase